src/main/xqy/qc-workspace-exporter.xqy

```diff
@@ -0,0 +1,54 @@
+xquery version "1.0-ml";
```

Comment:
I think these xqy modules need to be inline Strings in Java. When ml-javaclient-util is packaged up as a jar, it won't include these xqy modules. They'll need to be loaded from the classpath, so they could go into e.g. src/main/resources/ml-javaclient-util/. But for now, I recommend just inlining them as strings in Java. It's not pretty, but it's very simple from a packaging and deployment perspective.